api/server/router/container/container.go

```diff
@@ -60,6 +60,7 @@ func (r *containerRouter) initRoutes() {
 		// PUT
 		local.NewPutRoute("/containers/{name:.*}/archive", r.putContainersArchive),
 		// DELETE
+		local.NewDeleteRoute("/containers/_all", r.deleteAllContainers),
```

Comment:
@thaJeztah @tiborvass I have started making some changes to my PR, and am considering moving the container delete all method down to the daemon and making it faster. Does adding something like `containerRmAll` in `daemon/delete.go` make sense?